api/src/certification/complementary-certification/infrastructure/repositories/complementary-certification-badge-repository.js

```diff
@@ -41,6 +41,7 @@ const findAttachableBadgesByIds = async function ({ ids }) {
   const badges = await knex
     .from('badges')
     .whereIn('badges.id', ids)
+    .and.where('badges.isCertifiable', true)
```

Comment:
.where implique deja le .and
```suggestion
    .where('badges.isCertifiable', true)
```